src/legacy/core_plugins/interpreter/public/functions/markdown.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export default () => ({
+  name: 'kibana_markdown',
+  type: 'render',
+  context: {
+    types: [],
+  },
+  help: i18n.translate('common.core_plugins.interpreter.public.functions.markdown.help', {
+    defaultMessage: 'Markdown visualization'
+  }),
+  args: {
+    expression: {
+      types: ['string'],
+      aliases: [ '_' ],
+      default: '',
+      help: 'markdown',
```

Comment:
This text should probably contain more detail, as its presented to the viewer in the auto-complete.

Reply:
should, at some point. all the texts will need tobe worked on. at this stage we decided to keep them minimal/non existent